documents/motions/02.2018.1 - Definitions.md

```diff
@@ -137,7 +142,7 @@ Within WCA we use the following definitions for terminology.
     </tr>
     <tr>
       <td>Team</td>
-      <td>A team of the WCA that has been appointed by the WCA under the terms of these Bylaws or any of the Motions. Leaders of teams are WCA Staff and rest of team members are WCA External Staff.</td>
+      <td>A team of the WCA that has been appointed by the WCA under the terms of these Bylaws or any of the Motions. Leaders and Senior Members of teams are WCA Staff and rest of team members are WCA External Staff.</td>
```

Comment:
nit: In the definition of "Committee" earlier, you capitalized the `c`, do  you want to capitalize the the two `t`s here?

Also seems weird to be using the word "team" to define the term "team" here.